### DISCUSSION TITLE
Inline-style #card-reverse?

### DISCUSSION THREAD

**Discussion by sorenfrederiksen** [reuseman/flashcards-obsidian#32] [Ideas]
Hi @reuseman,

I'm writing, mostly, to say I love this plugin and really appreciate you putting in the hard work to make this happen.

(Right now, I'm studying for a citizenship and a related language test, and the ability to turn my notes for both quickly and painlessly into flashcards has been amazing. For me, this is the "killer app" plugin in the Obsidian community and has really simplified a small but important part of my life.)

Anyway, as the title suggests, I've made widespread use of the "inline-style" card definition syntax. The other types of cards are also great, but the inline is perfect for my "Some word I'm learning :: And its definition" use-case. The only thing this plugin is missing, from my point of view, is the ability to create double-sided "#card-reverse" cards using this kind of inline syntax.

Perhaps it could be something as simple as:

```markdown
Side A of double-sided card ::: Side B of doubled-sided card
```
Or maybe:

```markdown
Side A of double-sided card :.: Side B of double-sided card
```

It would be just great for my language learning use-case.

Is this something that's already possible? Have I misunderstood the documentation?

If it's easy to implement and you find yourself with some time spare time, it'd be great to see. If not, thanks again for helping us all out with this plugin.

**Comment by reuseman** [comment]
Hey @sorenfrederiksen, it took me a while to find the time, but now this is up and working. I opted for your first proposal, it's clean and does not interrupt the flow while writing. Hope it will be useful for you :smiley:

**Reply by sorenfrederiksen** [reply]
Thanks so much @reuseman! I really appreciate you finding the time (and agree on the flow assessment). I've got a practice exam tomorrow and this has given my vocab deck a real boost! Thanks again and have a nice week! 😄 💯
